specs/integrations/synthetics.md

```diff
@@ -0,0 +1,42 @@
+## Synthetics Integration
+
+Synthetic monitors play a crucial role in periodically checking the status of your services and applications on a global scale. General documentation about synthetic monitors can be found in
+[Synthetics getting started page](https://www.elastic.co/guide/en/observability/current/synthetics-get-started.html).
+
+This integration goes in to more detail about how the sythetics monitors would
+be correlated with the APM traces. Synthetics traces can be categorized in to two
+main types
+  1. HTTP checks - These have one-one mapping with APM transactions
+  2. Browser checks - These have a one-to-many mapping with APM transactions
+
+### Correlation
+
+The Synthetics agent takes the responsibility of creating the [`traceparent`](../agents/tracing-distributed-tracing.md#trace_id-parent_id-and-traceparent) header for each outgoing network request associated with a test during every monitor execution.
+
+- `trace.id` and `parent.id`
+  - outgoing requests that are being explicity traced by the synthetics agent
+     will have the `parent.id` and `trace.id` as part of the trace context.
+  - must be unique for each step for a browser monitor
+  - must be unique for a http monitor
+- `sampled` Flag
+  - used to control the sampling decision for all the downstream services.
+  - 100% sampling when tracing is enabled
+
+These correlation values would be applicable even if the downstream services are traced by
+OpenTelemetry(OTEL)-based agents.
+
+### Identifying Synthetics trace
+
+Synthetics monitor executions creates `rootless traces` as these traces are not
+reported to the APM server. To overcome this limitation on the APM UI, we need
```

Comment:
@sqren do you think that would be a good enough indication? Is it even crucial that we guard against rootless traces?

Reply:
yeah, while not ideal I think this is do-able.

> Is it even crucial that we guard against rootless traces?

I used to think it was but I'm coming around. We only need the root trace if users want to see how time is spent in the browser. If users are ok with just being able to see transactions from backend agents that should still work. 

Eg for a trace like: `Service A -> Service B -> Service C`, where Service A is the browser the user will be able to see the trace as `Service B -> Service C`, which might be ok.

Reply:
I agree. So let's maybe remove the notion that rootless traces is something we need to overcome in the UI from the spec.

Long-term, I think it would be ideal if the synthetics network information was stored in the traces index, in a data model that's compatible with OTel. I think span events would probably allow us to do that. That's obviously beyond the scope of this spec.